<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneItemEdit</class>
 <widget class="QDialog" name="SceneItemEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SceneItemEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>700</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>646</width>
        <height>561</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nesne Adı :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditItemName"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAddNewImage">
         <property name="text">
          <string>Yeni Resim Ekle</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Attribute Tipi-İsmi :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAttrType"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAttrName"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxIsOutput">
           <property name="text">
            <string>Command id?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxAdditionalAttr">
           <property name="text">
            <string>Ek göster/kaybet özelliği</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ek Göster/Kaybet Attr İsmi :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAdditionalAttr"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAdditionalAttrIsCommand">
           <property name="text">
            <string>Command id?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="labelRotation">
           <property name="text">
            <string>Rotasyon   :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderRotationValue">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditRotationValue">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSpecialArea">
         <item>
          <widget class="QLabel" name="labelColorPreview">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFontSize">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1200000000</number>
           </property>
           <property name="value">
            <number>1200000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSelectColor">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBoxText"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTextContent"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonShowOnStart">
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>zValue[0]: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_zValue">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonOneCikar">
           <property name="text">
            <string>Öne Çıkart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonArkayaGetir">
           <property name="text">
            <string>Arkaya Getir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Nesneyi Sil</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="labelPrecision">
           <property name="text">
            <string>Virgülden Sonra Gösterilecek Rakam  Sayısı :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPrecision"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="comboBoxTextAlign">
           <item>
            <property name="text">
             <string>Align Center</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Align Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Align Right</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLineBreak">
           <property name="text">
            <string>Line-Break Olacak mı?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMaxUzunluk">
           <property name="text">
            <string>Maximum Uzunluk:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMaxUzunluk"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonShowHide">
         <property name="text">
          <string>Nesne Görünür,Görünmez Yap</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
